gbus/tx/sagastore.go

```diff
@@ -51,6 +54,7 @@ func (store *SagaStore) scanInstances(rows *sql.Rows) ([]*saga.Instance, error)
 		dec := gob.NewDecoder(reader)
 		var instance saga.Instance
 		instance.ConcurrencyCtrl = version
+		instance.CreatedAt = createdAt
```

Comment:
@kerenbe4 instance.CreatedAt is a time.Time so you will need to change this line to the following
if createdAt.Valid {
  instance.CreatedAt = createdAt.Time
}

Reply:
@rhinof I'm still getting errors for this one
The data in the new column is guaranteed not to be null, so I kept the `time.Time` type. but still, the app fails to scan it correctly to time.Time type
any idea why? 
BTW in stackoverflow it is recommended to add the `parseTime=true` to the connection string, I don't think it's a problem to add that param to the connection string
https://stackoverflow.com/questions/45040319/unsupported-scan-storing-driver-value-type-uint8-into-type-time-time
https://stackoverflow.com/questions/29341590/how-to-parse-time-from-database

Reply:
The problem of automatically adding the parameter to the connection string is that it may affect the user queries performed using the active transaction ( invocation.Tx() ) and may cause them problems when scanning the row set. (assuming they do not add the parameter and want to scan their date to a byte[] type if you would add the flag they can get a scan error which I do not think grabbit should impose such a constraint).

The stackoverflow posts are written from a application developer standpoint and not from a framework/library standpoint.

As for scanning into the mysql.NullTime struct, what error are you receiving ?

Reply:
@kerenbe4 ok I see the problem, see my comment 
https://github.com/wework/grabbit/pull/224/files/1cd5f58eec4d7e53300d2833748f1e71b196f367#r347927802